protocol/communities/community.go

```diff
@@ -1296,8 +1296,8 @@ func (o *Community) toBytes() ([]byte, error) {
 		return nil, ErrNotAdmin
 	}
 
-	//We are not owner or admin, use the received serialized version
-	if !o.IsOwnerOrAdmin() {
+	// If we are not a control node, use the received serialized version
+	if !o.IsOwner() {
```

Comment:
Maybe it is worth to rebase on top with https://github.com/status-im/status-go/pull/3772 or at least make @MishkaRogachev aware.

```suggestion
	if !o.IsControlNode() {
```

Reply:
@MishkaRogachev please change to control node on your PR, please

Reply:
ops, i've just merged this pr

Reply:
will be fixed by me in the scope of my other task